tests/test_one_client_server.py

```diff
@@ -298,15 +275,11 @@ async def test_read_write(self) -> None:
                     else:
                         raise
 
-    # TODO DM-37477: modify this test to expect a TypeError (and construct
-    # the OneClientServer directly, instead of calling create_server)
-    # once we drop support for sync connect_callback
     async def test_sync_connect_callback(self) -> None:
```

Comment:
Should this be renamed to test_async*?

Reply:
I added that sentence to the doc string of the test case.